tests/lost_reset/test

```diff
@@ -78,6 +90,12 @@ while ($line = <$fh_out>) {
 ok($found_msg, 1); # Was the message well-formed?
 ok($result_lost == $lost); # Do the two lost values agree?
 
+if (!$result_lost || !$lost || $result_lost != $lost) {
```

Comment:
It is silent when the test passes.  It only provides more information upon failure.